dokku

```diff
@@ -55,6 +55,8 @@ export DOKKU_EVENTS_LOGFILE=${DOKKU_EVENTS_LOGFILE:="$DOKKU_LOGS_DIR/events.log"
 export DOKKU_CONTAINER_LABEL=dokku
 export DOKKU_GLOBAL_RUN_ARGS="--label=$DOKKU_CONTAINER_LABEL"
 
+export DOKKU_VERSION="0.12.13"
```

Comment:
We should grab this from the file, not hardcode it here.

Reply:
Otherwise you need to handle a sed call to get this updated in the `contrib/release` process. I don't personally want to do that right now - maybe in the future if we turn the `dokku` bin into golang, at which point it can be done at build-time.

Reply:
I've already updated the replacement :) -> https://github.com/dokku/dokku/pull/3305/files#diff-b9f5212d14fb13cff4e50a60368ff8a2R198

I agree that go does is beautifully with compile type definitions though.

Reply:
I would really rather we don't do this for now. Can you revert this part of the change (and anything related)?

Reply:
reverted